src/containers/shared/components/Transaction/XChainCreateClaimID/Simple.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import Account from '../../Account'
+import { TransactionSimpleComponent, TransactionSimpleProps } from '../types'
+import { SimpleRow } from '../SimpleRow'
+import { XChainBridge } from '../XChainBridge'
+
+export const Simple: TransactionSimpleComponent = (
+  props: TransactionSimpleProps,
+) => {
+  const { t } = useTranslation()
+  const {
+    data: {
+      instructions: {
+        lockingDoor,
+        lockingIssue,
+        issuingDoor,
+        issuingIssue,
+        signatureReward,
+        otherChainAccount,
+        bridgeOwner,
+      },
+    },
+  } = props
+
+  return (
+    <>
+      <XChainBridge
+        lockingDoor={lockingDoor}
+        lockingIssue={lockingIssue}
+        issuingDoor={issuingDoor}
+        issuingIssue={issuingIssue}
+        signatureReward={signatureReward}
+        bridgeOwner={bridgeOwner}
+      />
+      <SimpleRow label={t('other_chain_account')}>
+        <Account account={otherChainAccount} link={false} />
```

Comment:
Is this not a link because the account does not exist in this network?

Reply:
Correct - it's referring to an account on the chain on the other side of the bridge